templates/esm/typescript_gapic/package.json

```diff
@@ -55,9 +55,10 @@
     "test:esm": "c8 mocha build/esm/test",
     "test": "npm run test:cjs && npm run test:esm",
     "compile:esm": "tsc -p ./tsconfig.esm.json",
-    "babel": "babel esm --out-dir build/cjs --extensions \".ts\" --out-file-extension .cjs --copy-files",
+    "babel": "babel esm --out-dir build/cjs --ignore \"esm/**/*.d.ts\" --extensions \".ts\" --out-file-extension .cjs --copy-files",
```

Comment:
Note that these changes will only apply to new libraries, we'll need to make these changes separately for `package.json` files for existing libraries.